app/src/main/java/com/asfoundation/wallet/Logger.java

```diff
@@ -3,4 +3,6 @@
 public interface Logger {
```

Comment:
what is this extension? .java?

![image](https://user-images.githubusercontent.com/3799625/78890432-8fb87880-7a5d-11ea-9819-1ec48ba887f7.png)

Reply:
I'll leave that change for the future ticket :p